### ISSUE TITLE
The code for generating predefined poses

### ISSUE THREAD

**Issue by wendaizhou** [nv-nguyen/template-pose#6]
This code was very helpful to me. I see the *.npy file in ./lib/poses/predefined_poses. However, there seems to be nothing in the code about how these poses are generated. Maybe the author can share this part of the code?

**Comment by nv-nguyen** [comment]
Hi @wendaizhou,

I used the predefined poses from [Learning Descriptors](https://arxiv.org/pdf/1502.05908.pdf). 

I added the code to generate predefined poses at a dense level (2562 viewpoints) with Blender [here](https://github.com/nv-nguyen/template-pose/tree/main/lib/poses/predefined_poses). 

However, there is a minor issue (it might be from Blender's version) since I can reproduce perfectly predefined poses at a dense level (2562 viewpoints) but not the ones at a coarse level (there are a few more viewpoints with Blender's version 2.82 that I am using now).

Hope it will be helpful!

--Nguyen
